src/plugins/rollbar.js

```diff
@@ -0,0 +1,46 @@
+import { version } from '_/package.json';
+
+import Rollbar from 'rollbar';
+
+// https://docs.rollbar.com/docs/rollbarjs-configuration-reference#notifier
+const config = {
+  accessToken: import.meta.env.VITE_ROLLBAR_ACCESS_TOKEN,
+  // Anonymize IP addresses
+  captureIp: 'anonymize',
+  // Capture uncaught exceptions and unhandled rejections
+  captureUncaught: true,
+  captureUnhandledRejections: true,
+  // Only allow items to be sent from certain domains
+  hostSafeList: [
+    'cuttle.cards',
+    ...(!import.meta.env.PROD) ? [
+      'localhost',
+    ] : [],
+  ],
+  payload: {
```

Comment:
From the docs, it looks like this info belongs in `context` rather than `payload` since it is more about the environment than the specific error, right?

Reply:
They actually have it in the payload
```
// Set the environment, default log level and the context
Rollbar.configure({logLevel: 'info', payload: {environment: 'staging', context: 'home#index'}}
```

I think what it's doing is seeding the default payload contents for errors that come through. Like eventually we might want to add username, and that would be something we'd add payload specific so we could reach out to the user and ask more about the issue or something.